package.json

```diff
@@ -24,14 +31,26 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "istanbul": "^0.3.2",
-    "jshint": "^2.5.10",
-    "mocha": "^2.0.1",
-    "should": "^4.1.0"
+    "@swc/cli": "^0.1.57",
+    "@swc/core": "^1.2.220",
+    "@swc/jest": "^0.2.22",
+    "@types/debug": "^4.1.7",
```

Comment:
This version does not match the installed `debug` version. Try to install the latest version of `debug` and verify that it still works as before.